web/src/site-admin/externalServices.tsx

```diff
@@ -0,0 +1,348 @@
+import { FormattingOptions } from '@sqs/jsonc-parser'
+import { setProperty } from '@sqs/jsonc-parser/lib/edit'
+import { flatMap, map } from 'lodash'
+import AmazonIcon from 'mdi-react/AmazonIcon'
+import BitbucketIcon from 'mdi-react/BitbucketIcon'
+import GithubCircleIcon from 'mdi-react/GithubCircleIcon'
+import GitIcon from 'mdi-react/GitIcon'
+import GitLabIcon from 'mdi-react/GitlabIcon'
+import React from 'react'
+import awsCodeCommitSchemaJSON from '../../../schema/aws_codecommit.schema.json'
+import bitbucketServerSchemaJSON from '../../../schema/bitbucket_server.schema.json'
+import githubSchemaJSON from '../../../schema/github.schema.json'
+import gitlabSchemaJSON from '../../../schema/gitlab.schema.json'
+import gitoliteSchemaJSON from '../../../schema/gitolite.schema.json'
+import otherExternalServiceSchemaJSON from '../../../schema/other_external_service.schema.json'
+import phabricatorSchemaJSON from '../../../schema/phabricator.schema.json'
+import { PhabricatorIcon } from '../../../shared/src/components/icons'
+import * as GQL from '../../../shared/src/graphql/schema'
+import { EditorAction } from './configHelpers.js'
+
+/**
+ * Metadata associated with a given external service.
+ */
+export interface ExternalServiceCategory {
```

Comment:
The name `ExternalServiceCategory` evokes an enum or union of all possible external service categories to me, not metadata associated with a given external service.

```suggestion
export interface ExternalServiceMetadata {
```